src/components/ui/input-validation-error/input-validation-error.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+type Props = {
+  inputType: 'email' | 'password' | 'confirm_password';
+};
+
+const errors: { [key: string]: string } = {
+  email: 'You have to pass correct email adress',
+  password: 'At least one big, one small letter and one nember, 6-15 symbols',
+  confirm_password: 'Must match with password',
+};
+
+export const InputValidationError = ({ inputType }: Props) => (
+  <p className="text-xs text-red-500 flex items-center mt-2">
+    <svg
```

Comment:
don't use svg like this, just import from assets/svg
`import ErrorIcon from 'assets/icons/error.svg`

Reply:
I still cant pass color to the svg natively

Reply:
fill="" or fill="currentColor"